libmscore/score.cpp

```diff
@@ -1855,7 +1855,7 @@ void MasterScore::removeExcerpt(Excerpt* ex)
 //   clone
 //---------------------------------------------------------
 
-MasterScore* MasterScore::clone()
+MasterScore* Score::clone()
```

Comment:
You cannot clone `Score` and get `MasterScore`. `MasterScore` inludes number of `Scores` which are excerpts as well as `MasterScore` is derived from `Score`. Except this, other changes look correct, so let's find the better way here and I'll merge the PR.

Reply:
This is my first contrubution, so I am not quite sure about the process.
Do I keep this PR and branch, or close and make a new?

If I keep, I guess it would be like:
revert commit, create new solution, combine commits, rebase and publish.
Ok?

Reply:
You can commit changes to this PR, then squash two commits and force push to this PR. I'll give you specific instructions when I won't be on mobile :)

Reply:
You can commit changes to this PR, then squash two commits and force push to this PR. I'll give you specific instructions when I won't be on mobile :)

Reply:
So, I have a new solution for review.
Commits are squashed into one, but branch not pushed yet.
What is next step?

Reply:
Git checkout YourBranchName
Git push -f origin YourBranchName